pom.xml

```diff
@@ -0,0 +1,344 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.smartregister</groupId>
+
+    <artifactId>opensrp-server-referral</artifactId>
+    <packaging>jar</packaging>
+
+    <version>1.0-Test-SNAPSHOT</version>
+
+
+    <name>opensrp-server-referral</name>
+    <description>OpenSRP Server Referral module</description>
+    <url>http://github.com/OpenSRP/opensrp-server-referral</url>
+
+    <properties>
+        <main.basedir>${project.basedir}</main.basedir>
+        <motech.version>0.11</motech.version>
+        <spring.version>3.1.0.RELEASE</spring.version>
+        <opensrp.updatePolicy>always</opensrp.updatePolicy>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <nexus-staging-maven-plugin.version>1.5.1</nexus-staging-maven-plugin.version>
+        <opensrp.connector.version>1.0.6-SNAPSHOT</opensrp.connector.version>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>nexus-snapshots</id>
+            <name>Nexus Snapshots Repository</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <uniqueVersion>false</uniqueVersion>
+        </snapshotRepository>
+        <repository>
+            <id>nexus-releases</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+        </repository>
+    </distributionManagement>
+
+    <scm>
+        <connection>scm:git:git@github.com:OpenSRP/opensrp-server-referral.git</connection>
+        <developerConnection>scm:git:git@github.com:OpenSRP/opensrp-server-referral.git</developerConnection>
+        <url>https://github.com/OpenSRP/opensrp-server-referral</url>
+    </scm>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+            <comments>Open Smart Register Platform (OpenSRP) is Licensed under the Apache License, Version 2.0
+            </comments>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>opensrp_developer</id>
+            <name>OpenSRP Developer</name>
+            <email>info@smartregister.org</email>
+            <url>https://groups.google.com/forum/#!forum/opensrp-dev</url>
+            <organization>OpenSRP</organization>
+            <organizationUrl>http://smartregister.org</organizationUrl>
+            <roles>
+                <role>developer</role>
+            </roles>
+            <timezone>Africa/Nairobi</timezone>
+        </developer>
+    </developers>
+
+    <repositories>
+        <repository>
+            <id>motech.opensrp</id>
```

Comment:
Please update this bit and add  [this](https://github.com/OpenSRP/opensrp-server-core/blob/766e5bf078fde0673bad4ec5957d67a5b55f6912/pom.xml#L73) to make sure you can load the `opensrp-connector`  from `sonatype` [here](https://oss.sonatype.org/content/groups/public/org/smartregister/opensrp-server-connector/)